src/test/ui/block-result/unexpected-return-on-unit.stderr

```diff
@@ -6,11 +6,11 @@ error[E0308]: mismatched types
    |
    = note: expected type `()`
               found type `usize`
-help: did you mean to add a semicolon here?
+help: try adding a semicolon
```

Comment:
Or does that look funny? The double `:` (after `help`, and again at the end) is a bit odd I guess.